rnote-ui/src/workspacebrowser/mod.rs

```diff
@@ -531,4 +553,20 @@ impl WorkspaceBrowser {
         // current workspace index
         self.select_workspace_by_index(current_workspace_index);
     }
+
+    pub fn parent_path(&self) -> Option<PathBuf> {
```

Comment:
I think this method is named wrong, it is not giving back the parent path. And there is already the `current_selected_workspace_row()` method for the same functionality, so I think you should use that

Reply:
> And there is already the selected_workspace_dir() for the same functionality, so I think you should use that

Oh I missunderstood the term "selected_workspace". I thought that this would return the path to the selected entry in the workspace-browser.

Reply:
no worries, it is somewhat ambiguous